### ISSUE TITLE
incorrect fortinet password removal

### ISSUE THREAD

**Issue by ryanmerolle** [intentionet/netconan#151]
fortinet passwords are encrypted in config, using the following syntax:
```
set password ENC <hash>
```

Netconan outputs the following after it anonymizes the password:
```
set password netconanRemoved0 <hash>
```

The has is not removed by the ENC syntax is (IE this is just stating Encrypted hash to follow.

**Comment by dhalperi** [comment]
Thanks @ryanmerolle - we'll fix this right off!

**Comment by dhalperi** [comment]
We shipped netconan 0.12.1 with this fix: https://pypi.org/project/netconan/0.12.1/
